api/target/kusttarget_test.go

```diff
@@ -183,6 +186,160 @@ func TestResources(t *testing.T) {
 	}
 }
 
+// TestRawResources tests that if a directory of raw Resources is given, the Resources
+// will be parsed
+func TestRawResources(t *testing.T) {
+	// use a real FS -- walk doesn't work on in-memory FS, and in-memory FS doesn't
+	// test windows specific behaviors
```

Comment:
Why doesn't work for in-memroy FS. I see that `Walk` is implemented for `fsInMemory`. It is supposed to work.

Reply:
It doesn't :)

Reply:
(because it has no tests)